openreview/agora/agora.py

```diff
@@ -0,0 +1,442 @@
+from .. import openreview
+
+import os
+import json
+
+class Agora(object):
+
+    def __init__(self, client, support_group_id, superuser, editor_id):
+
+        venue_group = openreview.Group(
+            id='-Agora',
+            readers=['everyone'],
+            writers=[support_group_id],
+            signatures=[support_group_id],
+            signatories=[],
+            members=[],
+
+        )
+        client.post_group(venue_group)
+
+        covid_group_id = '-Agora/COVID-19'
+        covid_group = openreview.Group(
+            id=covid_group_id,
+            readers=['everyone'],
+            writers=[support_group_id],
+            signatures=[support_group_id],
+            signatories=[],
+            members=[],
+        )
+
+        covid_editors = '{}/Editors'.format(covid_group_id)
+
+        header = {
+            "title": "Agora(ἀγορά) COVID-19",
```

Comment:
I'm not sure about including the greek word in the title of this venue... I think we should have a group page for the parent group (-Agora) that includes it along with an explanation. Adding it here seems a little confusing to me.

Reply:
I agree, we need a home page for Agora to explain the site and browse the venues by topic.

Reply:
Can we add a space between "Agora" and "(ἀγορά)"

Reply:
done